frame/vesting/src/lib.rs

```diff
@@ -233,10 +234,13 @@ decl_module! {
 		/// Emits `VestingCreated`.
 		///
 		/// # <weight>
-		/// - Creates a new storage entry, but is protected by a minimum transfer
-		///	   amount needed to succeed.
+		/// - `O(1)`.
+		/// - DbWeight: 3 Reads, 3 Writes
+		///     - Reads: Vesting Storage, Balances Locks, Target Account, [Sender Account]
+		///     - Writes: Vesting Storage, Balances Locks, Target Account, [Sender Account]
+		/// - Benchmark: 263 µs (min square analysis)
 		/// # </weight>
-		#[weight = SimpleDispatchInfo::FixedNormal(1_000_000_000)]
+        #[weight = 260_000_000 + T::DbWeight::get().reads_writes(4, 4)]
```

Comment:
we shouldn't count sender account I think
```suggestion
		#[weight = 260_000_000 + T::DbWeight::get().reads_writes(3, 3)]
```

Reply:
Spaces as well.